src/components/Form/Form.stories.tsx

```diff
@@ -79,6 +79,8 @@ export const Playground = (): ReactElement => {
 
 	const showState = boolean("Show state", false);
 	const prepopulate = boolean("Prepopulate", false);
+	const defaultValuesKnob = select("Default Values", ["None", "Has Defaults"], "None");
+	const showGetFormValues = select("GetFormValues", ["None", "Returns Undefined", "Returns Data"], "Returns Data");

```

Comment:
From the ticket: "Ensure that there are a few more fields with defaultValue than their are fields filled by getFormValues. We need this to prove that defaultValue is not being used when getFormValues exists and returns data." In line 92, please delete some of the entries and also I'd suggest changing the text in some of them to include the word "getFormValues"